stable/spring-cloud-data-flow/README.md

```diff
@@ -101,6 +127,19 @@ The following tables list the configurable parameters and their default values.
 
 ### MySQL Configuration
 
-| Parameter                  | Description           | Default                   |
-| -------------------------- | --------------------- | ------------------------- |
-| mysql.mysqlDatabase        | MySQL database name   | dataflow
+| Parameter                  | Description                  | Default                   |
+| -------------------------- | ---------------------------- | ------------------------- |
+| mysql.enabled              | Enable deployment of MySQL   | true
+| mysql.mysqlDatabase        | MySQL database name          | dataflow
+
+### External Database Configuration (mysql.enabled=false)
+
+| Parameter           | Description                    | Default                   |
+| ------------------- | ------------------------------ | ------------------------- |
+| database.driver     | Database driver                | org.mariadb.jdbc.Driver
+| database.host       | Database host                  | ""
+| database.port       | Database port                  | 3306
+| database.user       | Database user                  | scdf
+| database.password   | Database password              | ""
+| database.server     | Database name for SCDF server  | dataflow
```

Comment:
the name `database.server` sounds like its as it says -- the "database server". i think this could be clarified by say `database.scdf`, `database.dataflow`, etc as with `database.skipper`